### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by kazigk**
I hope that this translation is ok. How I can test my translation? I tried building this app and running it with locale set to "pl_PL", but the app was still in English.

Resolves issue #173

### CODE REVIEW

**Comment by johnjohndoe** [comment]
Thank you. I will review it soon. The app should show up in Polish once you check your your branch, build any product flavor and set your phones system language to Polish.

File: app/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (johnjohndoe) [inline app/src/main/res/values-pl/strings.xml]: Please keep the xliff namespace here: `<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">` and keep the xliff usage as in the English strings.xml file.

**Comment by kazigk** [comment]
Yes, I do want to complete the remaining ones. Please expect updates (commits) from me during this weekend, thanks :)

**Comment by johnjohndoe** [comment]
@kazigk Please let me know if you got stuck somewhere. I am happy to help.

**Comment by johnjohndoe** [comment]
@kazigk Do you still have time to complete your work? Can I help somehow?

**Comment by johnjohndoe** [comment]
@kazigk Is there any chance you will have time to finish the translation?

**Comment by johnjohndoe** [comment]
Closes because stale since Oct 31, 2019.

**Comment by johnjohndoe** [comment]
Happy new year @kazigk! I started managing translations for the EventFahrplan at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you to join there if you like.

**Pull request closed without merging**